jetstream/message.go

```diff
@@ -75,6 +75,15 @@ type (
 		// Term tells the server to not redeliver this message, regardless of
 		// the value of MaxDeliver.
 		Term() error
+
+		// TermWithReason tells the server to not redeliver this message, regardless of
+		// the value of MaxDeliver. The provided reason will be included in JetStream
+		// advisory event sent by the server.
+		//
+		// Note: This will only work with JetStream servers >= 2.10.4.
```

Comment:
But `ackReply` does a simple publish on a JS reply subject, it does not wait for server ack (only `DoubleAck` does that). So it would simply work as if the message was not terminated and will eventually be redelivered.

Reply:
Ah, i always get confused with this method name, you're right.